layout/_partial/sidebar-footer_image.ejs

```diff
@@ -0,0 +1,9 @@
+<% if(theme.img.footer_image) { %>
+    <div id="footer-image">
```

Comment:
Hmmm...
In one div block it's horizontal so multiple pictures show well.
In multiple div block it's vertical and only show one picture. The next one have to be scroll to show.
I will try to fix it although I have no idea what is leading to such a strange behavior.